addons/website_event_sale/views/website_event_templates.xml

```diff
@@ -18,19 +18,16 @@
             <t t-if="(ticket.price - ticket.price_reduce) &gt; 1 and website.pricelist_id.discount_policy == 'without_discount'">
                 <del class="text-danger me-1" t-field="ticket.price" t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.pricelist_id.currency_id}"/>
             </t>
-            <span t-field="ticket.price_reduce"
-                  t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.pricelist_id.currency_id}"
-                  groups="account.group_show_line_subtotals_tax_excluded"/>
-            <span t-field="ticket.price_reduce_taxinc"
-                  t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.pricelist_id.currency_id}"
-                  groups="account.group_show_line_subtotals_tax_included"/>
-            <span itemprop="price" class="d-none" t-out="ticket.price"/>
-            <span itemprop="priceCurrency" class="d-none" t-out="website.pricelist_id.currency_id.name"/>
+            <t t-set="display_price" t-value="ticket.price_reduce" groups="account.group_show_line_subtotals_tax_excluded"/>
+            <t t-set="display_price" t-value="ticket.price_reduce_taxinc" groups="account.group_show_line_subtotals_tax_included"/>
+            <span t-out="display_price"
+                  t-options="{'widget': 'monetary', 'from_currency': event.company_id.sudo().currency_id, 'display_currency': website.pricelist_id.currency_id}"/>
```

Comment:
I would rather keep as close as possible to the original, purely to avoid breaking existing custom templates.

`@t-field="some_field"` is something that's very likely to be used in xpath